examples/Console/Fax/DownloadFax.cs

```diff
@@ -0,0 +1,15 @@
+using Sinch;
+
+namespace Examples.Fax
+{
+    public class DownloadFax
+    {
+        public static async Task Example()
+        {
+            var sinchClient = new SinchClient("PROJECT_ID", "KEY_ID", "KEY_SECRET");
+            await using var responseStream = await sinchClient.Fax.Faxes.DownloadContent("FAX_ID");
+            await using var fileStream = new FileStream("C:\\Downloads\\fax.pdf", FileMode.Create, FileAccess.Write);
```

Comment:
It can be worth to show how to read the file name from the response rather than hardcoding it

Reply:
Yes, it's worth to illustrate how to read the file name

Reply:
This is what I'm doing in Node.js: the return type is an object that contains the fileName extracted from the content-disposition header and the buffer. Can you elaborate why you consider this structure not that clean?
And it's true that the filename returned by the server is the faxId, but hardcoding it in the example seems a hack to compensate for the loss of information from the header.

Reply:
I'll just implement that